src/GenFacades/GenFacades.Console/GenFacades.Console.net46.csproj

```diff
@@ -4,25 +4,23 @@
 
   <PropertyGroup>
```

Comment:
I don't think it should be problematic. The output gets put into the "GenFacades.Console.net46" folder, and the project.json in the directory supports both targets. So it should be okay.